docs/source/AdministratorGuide/DIRACSites/index.rst

```diff
@@ -1,10 +1,52 @@
-===========================================
-Managing DIRAC Sites
-===========================================
+.. _managingSites:
+
+=====================================
+Managing Sites and Resources in DIRAC
+=====================================
+
+A Site, in DIRAC, is the entity that collects access points to resources that are related by locality in a functional sense,
+i.e. the storage at a given Site is considered local to the CPU at the same Site and this relation will be used by DIRAC.
+On the other hand, a Site must provide a single entry point responsible for the availability of the resources that it encompasses.
+In the DIRAC sense, a Site can be from a fraction of physical computer center, to a whole regional grid.
+It is the responsibility of the DIRAC administrator of the installation to properly define the sites.
+Not all Sites need to grant access to all VOs supported in the DIRAC installation.
+
+DIRAC can incorporate resources provided by existing Grid infrastructures (e.g. WLCG, OSG) as well as
+sites not integrated in any grid infrastructure, but still
+contributing with their computing and storage capacity, available as conventional clusters or file servers.
+
+
+Site Names
+==========
+
+In the :ref:`DIRAC configuration Sites <cs-site>` have names resulting from concatenation of the Domain prefix, the name of the Site and the country (or the funding body),
+according to the ISO 3166 standard with a dot as a separator. 
+The full DIRAC Site Name becomes of the form: [Domain].[Site].[co].
+The full site names are used everywhere when the site resources are assigned to the context of a particular Domain:
+in the accounting, monitoring, configuration of the Operations parameters, etc.
+
+Examples of valid site names are::
```

Comment:
```suggestion
Examples of valid site names are:
```

the two colons would make the bullet list be formatted like "verbatim", instead of like a bullet list.
The indentation for the bullet list is not necessary, I think it will result in an additional level.